CMakeLists.txt

```diff
@@ -148,11 +108,64 @@ set(CLP_FFI_JS_SRC_ZSTD
     src/submodules/zstd/lib/decompress/zstd_decompress.c
 )
 
-target_sources(
-    ${CLP_FFI_JS_BIN_NAME}
-    PRIVATE
+# List of supported environments.
+set(CLP_FFI_JS_ENVIRONMENTS
+    "node"
+    "worker"
+)
+
+foreach(env ${CLP_FFI_JS_ENVIRONMENTS})
+    set(CLP_FFI_JS_BIN_NAME "ClpFfiJs-${env}")
+    add_executable(${CLP_FFI_JS_BIN_NAME})
+    target_compile_features(${CLP_FFI_JS_BIN_NAME} PRIVATE cxx_std_20)
+    target_compile_options(${CLP_FFI_JS_BIN_NAME} PRIVATE ${CLP_FFI_JS_ENABLE_EXCEPTION_FLAG})
```

Comment:
Do we need to [add](https://emscripten.org/docs/optimizing/Optimizing-Code.html?highlight=lto#lto) `-flto`?

Reply:
> Thus, to allow maximal LTO opportunities with the LLVM Wasm backend, build all source files with -flto and also link with flto.

It seems so

Reply:
yeah sorry for the oversight: https://emscripten.org/docs/tools_reference/emcc.html#emcc-closure:~:text=%5Bcompile%2Blink%5D%20Enables%20link%2Dtime%20optimizations%20(LTO).